generators/client/templates/angular/src/main/webapp/app/shared/_shared-common.module.ts

```diff
@@ -18,6 +18,16 @@
 -%>
 import { NgModule, LOCALE_ID } from '@angular/core';
 import { Title } from '@angular/platform-browser';
+import { registerLocaleData } from '@angular/common';
+<%_ const localeIds = {
+    'zh-cn': 'zh-Hans',
```

Comment:
I suggest to add a new attribute to the original language object in generator-constants so that this is not hidden, then fetch the value from that object lile how we do for rtl

Reply:
done